civiclens/collect/move_data_from_api_to_database.py

```diff
@@ -833,7 +924,7 @@ def pull_all_api_data_for_date_range(
 
     if pull_comments:
         print("adding comments to the db")
-        add_comments_to_db(commentable_docs)
+        add_comments_to_date_range(start_date, end_date)
```

Comment:
I think we should break this into a different function. I agree with the creation of this new function -- it's more intuitive. But I think we should call it differently so there's no confusion on how it's working under the hood

Reply:
Thinking about it a little further, maybe it's nit necessary to break into it's own function -- we just have to be sure to update the doc string clearly

Reply:
after seeing it again, it is bad naming. will discuss it later.

Regarding permanent/temporary existing data, I'm leaning toward making it permanent, especially for checking the open_to_comment update

Reply:
sounds good